### PR TITLE
feat: [3376] add new updates for full timezone support

### PR DESCRIPTION
**Pull request by sayinmehmet47**
tested it in local works fine now with diff timezones. we can publish it

### CODE REVIEW

File: src/addons/dragAndDrop/backgroundWrapper.js [hunk 1]
```diff
@@ -50,46 +50,6 @@ const propTypes = {
 }
 
 class DraggableBackgroundWrapper extends React.Component {
-  // constructor(...args) {
```
Comment (damian-kociuba) [inline src/addons/dragAndDrop/backgroundWrapper.js:53]: are you sure you want to comment this component? I would uncomment it or delete if it isn't needed longer

File: stories/helpers/customComponents.js [hunk 2]
```diff
@@ -53,22 +53,22 @@ const customComponents = {
...
-            backgroundColor:
-              timeSlotWrapperProps.value.getHours() >= 8 &&
-              timeSlotWrapperProps.value.getHours() <= 17
-                ? 'green'
-                : 'red',
-            padding: '5px',
-          }
-    return <div style={style}>{timeSlotWrapperProps.children}</div>
-  },
+  // timeSlotWrapper: timeSlotWrapperProps => {
```
Comment (damian-kociuba) [inline stories/helpers/customComponents.js]: this commented section looks suspicious. Is it intended?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@healthadvisor/react-big-calendar",
-  "version": "1.0.4",
+  "version": "1.0.5",
```
Comment (mmoczulski) [inline package.json]: let's give it v.1.1.0

File: src/addons/dragAndDrop/backgroundWrapper.js (hunk 1 above)
Comment (mmoczulski) [reply src/addons/dragAndDrop/backgroundWrapper.js:53]: Mehmet has remoed this comment 😛

**Review by mmoczulski — APPROVED** [review]

**Comment by mmoczulski** [comment]
and please add some comment to changelog for v1.1.0 entry

File: src/addons/dragAndDrop/backgroundWrapper.js (hunk 1 above)
Comment (sayinmehmet47) [reply src/addons/dragAndDrop/backgroundWrapper.js:53]: this was old comment , just i droped it

File: stories/helpers/customComponents.js (hunk 2 above)
Comment (sayinmehmet47) [reply stories/helpers/customComponents.js]: it was just a story , i dont think we use storybook. But i will uncomment it again

File: package.json (hunk 3 above)
Comment (sayinmehmet47) [reply package.json]: i was not sure if it is major change, then it is clear

**Review by mmoczulski — APPROVED** [review]
published https://www.npmjs.com/package/@healthadvisor/react-big-calendar v1.1.0

**Pull request merged**